lib/android.js

```diff
@@ -582,18 +582,24 @@ function tryDetectInstrumentationOffset () {
     cur = insn.next;
   }
 
+  if (getAndroidApiLevel() >= 30) {
+    throw new Error('Unable to determine Runtime.instrumentation_ offset');
+  }
```

Comment:
I do like this idea, but it's probably better that we only fail when attempting to use `Java.deoptimizeEverything()`, so we're (best-case) only partially broken when running on future Android releases.

Reply:
Ah, nice catch. I agree with your opinion, and I guess line 1770 is enough for catching the problem.